Common/src/main/java/dk/sdu/se4/common/service/AIControlSystem.java

```diff
@@ -10,5 +10,8 @@
  * @author steff
  */
 public interface AIControlSystem {
-    void pathFinding();
+
+    void addPathFinding();
+
+    void removePathFinding();
 }
```

Comment:
What does the add and remove methods do?

Reply:
adding the pathfinding and removing the partfinding module